source/Handlebars.Test/BasicIntegrationTests.cs

```diff
@@ -37,6 +37,13 @@ public class HandlebarsEnvGenerator : IEnumerable<object[]>
     
     public class BasicIntegrationTests
     {
+        private static string HtmlEncodeStringHelper(IHandlebars handlebars, string inputString)

```

Comment:
As you've introduced helper method to get `expected` value it might be reasonable to add new entry to `HandlebarsEnvGenerator` to run tests with new encoder as well.

Reply:
Added Handlebars with new `HtmlEncoder` to `HandlebarsEnvGenerator`.